components/browser/engine-system/src/test/java/mozilla/components/browser/engine/system/SystemEngineSessionTest.kt

```diff
@@ -633,7 +637,8 @@ class SystemEngineSessionTest {
     @Test
     fun desktopMode() {
```

Comment:
There's one more test that needs an update: `withProvidedDefaultSettings` to test that an engine default can be specified for `useWideViewPort`.